docs/src/pages/demos/avatars/avatars.md

```diff
@@ -1,3 +1,7 @@
+---
+avatar
```

Comment:
I would add a description, to follow bootstrap approach
https://raw.githubusercontent.com/twbs/bootstrap/v4-dev/docs/about/history.md so it's displayed as a table.